Makefile

```diff
@@ -38,7 +39,7 @@ docs/telemeter_query: $(JSONNET_SRC)
 	echo "$$query" > $@
 
 test-generate:
-	make --always-make && git diff --exit-code
+	make manifests && git diff --exit-code
```

Comment:
We *have* to have `--always-make` here, otherwise we can't guarantee that the manifests will be re-generated for the test, in which case the test won't actually compare anything new.